pkg/cluster/cluster_config.go

```diff
@@ -95,6 +95,16 @@ func GetDomain(ctx context.Context, c client.Client) (string, error) {
 	return domain, err
 }
 
+func GetOCPVersion(ctx context.Context, c client.Client) (semver.Version, error) {
```

Comment:
This can probably be computed at startup

Reply:
did some change, moved this call a bit up to the main